substrate/frame/message-queue/src/lib.rs

```diff
@@ -774,6 +776,15 @@ enum MessageExecutionStatus {
 	StackLimitReached,
 }
 
+/// The status of an attempt to process a message.
+#[derive(PartialEq)]
+enum ServiceQueuesContext {
+	/// There is not enough weight remaining at present.
+	OnIdle,
+	/// There will never be enough weight.
```

Comment:
These two too?!

Reply:
Yeah I copied the enum formatting and forgot to update docs. Pushed a new commit to fix.